roles/filetree_create/templates/current_credential_types.j2

```diff
@@ -2,14 +2,23 @@
 controller_credential_types:
 {% for credential_type in current_credential_types_asset_value %}
   - name: "{{ credential_type.name }}"
-    description: "{{ credential_type.description }}"
+    description: "{{ template_overrides_resources.credential_type[credential_type.name].description
+                     | default(template_overrides_global.credential_type.description)
+                     | default(credential_type.description) }}"
     kind: "{{ credential_type.kind }}"
     inputs:
-{{ credential_type.inputs | to_nice_yaml(indent=2) | indent(width=6, first=True) }}
-{% if credential_type.injectors and credential_type.injectors != '---' %}
+{{ template_overrides_resources.credential_type[credential_type.name].inputs
+   | default(template_overrides_global.credential_type.inputs)
```

Comment:
Makes it sense setting all the exported `credential_types` with the same (`template_overrides_global`) inputs?

Reply:
Almost certainly no for this. But it doesn't harm leaving the option I suppose

Reply:
It can lead to more errors than necessary... I'd prefer not to let this kind of modifications.